qt/python/mantidqt/widgets/commonworkspacedisplay/status_bar_view.py

```diff
@@ -0,0 +1,24 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+#
+#
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QMainWindow, QStatusBar
+
+
+class StatusBarView(QMainWindow):
+    def __init__(self, parent, central_widget, name):
+        super(StatusBarView, self).__init__(parent)
+        self.setCentralWidget(central_widget)
+        self.setWindowTitle("{} - Mantid".format(name))
+        self.setAttribute(Qt.WA_DeleteOnClose, True)
+
+        self.status_bar = QStatusBar(self)
+        self.setStatusBar(self.status_bar)
+
+        self.resize(600, 400)
```

Comment:
Just out of curiosity why is resize necessary?

Reply:
Qt seems to have some idea of the minimum size, but that appears pretty small on 1920x1080. Giving a size of 600x400 makes them pretty much the same size as Mantid Plot and I like the consistency between Mantid Plot and the Workbench

Reply:
:+1: for the consistency. Should we make the default size a constructor parameter so that it can be easily overridden so it wouldn't have to be resized twice?